ext/gd/libgd/gd.c

```diff
@@ -1134,7 +1134,7 @@ void gdImageLine (gdImagePtr im, int x1, int y1, int x2, int y2, int color)
 TBB: but watch out for /0! */
 			double ac = cos (atan2 (dy, dx));
 			if (ac != 0) {
-				wid = thick / ac;
+				wid = (int) (thick / ac);
```

Comment:
It seems to me that rounding would make sense here, but likely introduces a subtle BC break; as such casting to int is okay.